pkg/fission-cli/flag/flag.go

```diff
@@ -76,6 +76,8 @@ var (
 
 	KubeContext = Flag{Type: String, Name: flagkey.KubeContext, Usage: "Kubernetes context to be used for the execution of Fission commands", DefaultValue: ""}
 
+	IgnoreNotFound = Flag{Type: Bool, Name: flagkey.IgnoreNotFound, Usage: "Avoid error if Fission resource not found during deletion", DefaultValue: false}
```

Comment:
Please check usage to .
```
Treat "resource not found" as a successful delete
```
Reference https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete